rmgpy/molecule/graph.pyx

```diff
@@ -1043,11 +1057,13 @@ cdef class Graph:
         cdef bint selfHasEdge, otherHasEdge
         cdef int i, j
         
-        # Check that the mapped pairs of vertices are equivalent
+        method = 'equivalent' if equivalent else 'isSpecificCaseOf'
+        
+        # Check that the mapped pairs of vertices compare True
         for vertex1, vertex2 in mapping.items():
-            if not vertex1.equivalent(vertex2):
+            if not getattr(vertex1,method)(vertex2):#not vertex1.isSpecificCaseOf(vertex2):
```

Comment:
Was the comment (and the one in line 1078) left intentionally?

Reply:
Nope, should be fixed now.